### ISSUE TITLE
2.4.10 would not install in Linux Mint 22 

### ISSUE THREAD

**Issue by rcentros** [trelby/trelby#27]
I built a new Linux Mint 22 computer and tried to install Trelby 2.4.10 via  the .deb file -- it installed but failed to run. It's the package found via the SUSE site. (Maybe the issue comes from there?) At any rate this is the error I get, if it makes any sense to you.

```
Traceback (most recent call last);
   File "/usr/bin/trelby", line 9, in <module>
       import trelby
ModuleNotFoundError: No module named 'trelby'
```
Fortunately I still had a copy of the 2.4.9 .deb file on another computer and was able to install it. Is it possible to put some of the releases between 2.2 and 2.4.10 for download here? (There used to be .deb packages available on some of the older releases on the old GitHub site.)

I'll get in touch with the folks at SUSE about this also. Thanks for all your work.

**Comment by txtsd** [comment]
The binary has been copied to `/usr/bin`. It should not be. Launch it from `/opt/trelby/bin/` or wherever it's installed. If you want, make a script that invokes it from `/usr/bin`.

**Comment by rcentros** [comment]
Thanks. The .deb file did install the trelby file in /usr/bin. The problem was that instead of referencing /usr/share/trelby/src they had entered into the trelby executable file the following... 

`/usr/share/trelby//usr/share/trelby//usr/share/trelby/src`

Of course that doesn't make any sense. Once I removed the two extraneous /usr/share/trelby entries, it worked as before. And, as before, it still did not call for 'python3-reportlab' — which I install on my own. I'm guessing Ubuntu must include that file by default where Linux Mint doesn't.

At any rate, it works now. Thanks for the pointer.

**Comment by UnisTorvalds** [comment]
This occurred to me as well on Mint 21.3
Trelby 2.4.9 worked fine.  Updated to 2.4.10 from GwynC's PPA.  Now won't launch.  Binary was copied to /usr/bin/trelby and /usr/share/trelby

**Comment by rcentros** [comment]
Just edit the trelby file in /usr/bin and remove the extra two **/usr/share/trelby** entries. Something went wonky when they made it into a .deb file.

**Comment by UnisTorvalds** [comment]
Solved. Thank you!

**Comment by limburgher** [comment]
I'll try to fix this for 2.4.11. I'm a very experienced RPM packager, but am still very green with DEB. :)

**Comment by limburgher** [comment]
Try the PyPi version: pip install trelby.

**Comment by limburgher** [comment]
Fixed with 2.4.12.
